pkg/loader/compose/compose.go

```diff
@@ -359,6 +359,10 @@ func (c *Compose) LoadFile(files []string) (kobject.KomposeObject, error) {
 			}
 		}
 
+		err = checkLabelsPorts(len(serviceConfig.Port), composeServiceConfig.Labels["kompose.service.type"], name)
+		if err != nil {
+			log.Fatal(err)
```

Comment:
Let's not use `log.Fatal`, instead wrap the error and pass it up the call stack. Use `errors.Wrap`